spec/turn_spec.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/turn'
+require './lib/card'
+
+RSpec.describe Turn do
+  it 'takes in string (guess) and card object' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+    
+    expect(turn.guess).to eq("Juneau")  
+    expect(turn.card).to eq(card)
+  end
+  
+  it "checks to see if the guess is correct" do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn.correct?).to be(true) 
```

Comment:
Same here. Rspec has built-in predicate matchers that we can take advantage of: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/predicate-matchers

```
expect(turn).to be_correct
```